### ISSUE TITLE
Ensure all Figma colors have enough contrast for WCAG standards

### ISSUE THREAD

**Issue by st0nebraker** [sourcegraph/sourcegraph#50190]
Even non-text graphical object need a minimum contrast of at least 3:1. Many of our semantic colors such as `info` violate that on light theme.

* https://webaim.org/resources/contrastchecker/ is a great tool in testing colors contrast levels & passibility.

We should ensure all our colors on light & dark themes pass WCAG color contrast standards.
